### ISSUE TITLE
Project is missing good roadmap (After meet#1) 

### ISSUE THREAD

**Issue by lil5** [gorgabal/EnvelopeBudget#6]
# Topics discussed

## 1. How to engineer backend
   1. query data with SQLite (easy query, hard to support modified files)
   1. parse data with Java (may be slower, too be written)
   1. query data with Java (may be slower, too be written)
   1. query and parce data run C++ ledger lib connected with Java (OS support?)

## 2. Which platform(s)
   1. android
         1. Desktop already have Ledger-cli, Android needs priority. 
         1. Data is necessary before accountancy is possible. 
         1. Focus on simplicity. (Ease of use)
   1. desktop(Linux,deb?)
         1. Must be able to use similar code base as Android.
         1. Focus on data accountancy, querys and (Y/M/W/...) reports.

**Comment by lil5** [comment]
1.4 Java connect C++
Android: https://developer.android.com/ndk/guides/index.html

Desktop:
http://en.wikipedia.org/wiki/Java_Native_Interface

**Comment by lil5** [comment]
2.2.1 Java connect with JavaScript (reactJS)

Android:
[Stackoverflow reactnative to java bridge explained](https://stackoverflow.com/questions/41527440/importing-a-native-bridge-module-into-javascript-in-react-native#)
https://facebook.github.io/react-native/

Desktop:
JavaFX WebView [Example](https://github.com/lipido/javafxwebview) http://docs.oracle.com/javafx/2/api/javafx/scene/web/WebView.html

See more at #1

**Comment by lil5** [comment]
@gorgabal
Could you test the C ++ language compatibility layers

I will be testing those of JavaScript

Will make a shared repo for reference later on

**Comment by gorgabal** [comment]
I don't think this is what we've agreed on. Could you send me a draft next time before publicly publishing one? I feel like we are misunderstanding each other big time. 

I will contact you later today.

**Comment by gorgabal** [comment]
Please add stuff like this on the wiki, it is not what issues are for.

**Comment by gorgabal** [comment]
After calling with @lil5 , the following become more clear. (Lucian, please correct me if I am wrong)

The issue is that there is not really a concrete enough road-map. This issue will be solved once a more concrete path of action has been defined.
